### PR TITLE
Merge from release/3.0

### PR DESCRIPTION
**Pull request by mmitche**

### CODE REVIEW

File: eng/common/performance/performance-setup.ps1 [hunk 1]
```diff
@@ -1,8 +1,9 @@
 Param(
     [string] $SourceDirectory=$env:BUILD_SOURCESDIRECTORY,
     [string] $CoreRootDirectory,
+    [string] $BaselineCoreRootDirectory,
     [string] $Architecture="x64",
-    [string] $Framework="netcoreapp3.0",
+    [string] $Framework="netcoreapp5.0",
```
Comment (phenning) [inline eng/common/performance/performance-setup.ps1]: netcoreapp3.1?

File: eng/common/performance/performance-setup.sh [hunk 2]
```diff
@@ -2,20 +2,23 @@
 
 source_directory=$BUILD_SOURCESDIRECTORY
 core_root_directory=
+baseline_core_root_directory=
 architecture=x64
-framework=netcoreapp3.0
+framework=netcoreapp5.0
```
Comment (phenning) [inline eng/common/performance/performance-setup.sh]: netcoreapp3.1?

File: eng/common/performance/performance-setup.ps1 (hunk 1 above)
Comment (mmitche) [reply eng/common/performance/performance-setup.ps1]: These are in eng/common.  Should be left as-is.

File: eng/common/performance/performance-setup.sh (hunk 2 above)
Comment (mmitche) [reply eng/common/performance/performance-setup.sh]: These are in eng/common. Should be left as-is.

**Pull request merged**